packages/EasyBugsnag/docs/aws.md

```diff
@@ -0,0 +1,19 @@
+---eonx_docs---
+title: 'AWS information'
+weight: 1005
+---eonx_docs---
+
+# AWS information
```

Comment:
We should be explicit about "AWS ECS Fargate" as the implementation works only for it.